sopel/tools/target.py

```diff
@@ -58,9 +58,10 @@ def __init__(
         self.away = None
         """Whether the user is marked as away."""
 
-    hostmask = property(lambda self: '{}!{}@{}'.format(self.nick, self.user,
-                                                       self.host))
-    """The user's full hostmask."""
+    @property
+    def hostmask(self) -> str:
+        """The user's full hostmask."""
+        return '{}!{}@{}'.format(self.nick, self.user, self.host)
```

Comment:
This is much better than `property(lambda self: …)`, nice catch!

Reply:
This overlaps with a change from #2448, actually. @Exirel & I same-brained this one. 😁